tracee-rules/engine/engine.go

```diff
@@ -210,12 +205,21 @@ func (engine *Engine) consumeSources(done <-chan bool) {
 	}
 }
 
-func (engine *Engine) dispatchEvent(s types.Signature, pe ParsedEvent) {
+func (engine *Engine) dispatchEvent(s types.Signature, event tracee.Event, parsedEvent bool) {
 	switch {
 	case strings.Contains(reflect.TypeOf(s).String(), "rego"):
-		engine.signatures[s] <- pe
+		if parsedEvent {
```

Comment:
I think now we can remove parsedEvent arg from method declaration and invocations:

```suggestion
		if engine.parsedEvents {
```

Reply:
sorry missed this one - fixed